### ISSUE TITLE
Connecting O365 using Modern Authentication

### ISSUE THREAD

**Issue by KishorekumarElangovanSecurrency** [OfficeDev/O365-ActivityFeed-AzureFunction#23]
This is connecting the exchange using basic authentication. Since the basic authentication is going to be deprecated by microsoft for O365 in oct'22, is there an alternate way to connect to O365 using modern authentication? If yes is it possible to use it in this code? Or is it possible to use Connect-ExchangeOnline module?

**Comment by jonnords** [comment]
Yes, you can replace the connection to EXO with a managed identity as an example. Simplest for now is likely to load the EXO modules and change the connection string for EXO.
